iothub/service/samples/PnpServiceSamples/TemperatureController/Microsoft.Azure.Devices.TemperatureController/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json;
+using PnpHelpers;
```

Comment:
By helper are just creating helper methods or a helper class? I probably did not quite catch what it meant in the meeting.

Reply:
Also I cannot find this file somehow. Can you point where it is?

Reply:
yes it's here: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/pnp/PnpHelper.cs

However this needs to be brought to preview